client/components/AddSponsor.tsx

```diff
@@ -29,32 +36,43 @@ const AddSponsor: React.FC = () => {
 
   return (
     <>
-      {(responseMsg || errors) && <ResponseDiv>{responseMsg}</ResponseDiv>}
-      <Form onSubmit={handleSubmit(onSubmit)}>
-        <label>Sponsor Name: </label>
-        <Input
+      {(responseMsg || errors) && <div>{responseMsg}</div>}
+      <form onSubmit={handleSubmit(onSubmit)} className={styles.form}>
+        <TextField
+          label="Sponsor Name"
           name="name"
           type="text"
           placeholder="Glitter and Sparkle Co"
           ref={register}
           required
+          className={styles.item}
         />
-        <label>Sponsor Website: </label>
-        <Input
+        <TextField
+          label="Sponsor Website"
           name="website"
           type="text"
           placeholder="www.glitter.co"
           ref={register}
           required
+          className={styles.item}
         />
-        <label>Sponsor Type: </label>
-        <select name="type" required ref={register}>
-          <option value="FOOD">Food</option>
-          <option value="BEVERAGE">Beverage</option>
-          <option value="OTHER">Other</option>
-        </select>
-        <SubmitBtn type="submit">Add Sponsor</SubmitBtn>
-      </Form>
+        <label className={styles.item}>
+          Sponsor Type:
+          <select name="type" required ref={register}>
```

Comment:
any reason to not use the material-ui `Select` component here?

Reply:
Great idea 🚀